File: 코딩 기초 트레이닝/수 조작하기1-p181926/수 조작하기1.js
// https://school.programmers.co.kr/learn/courses/30/lessons/181926

function solution(n, control) {
    const dic = {
        'w': 1,
        's': -1,
        'd': 10,
        'a': -10
    };

    let answer = n;
    control = control.split("");

    control.forEach((c) => {
        answer += dic[c];
    });

    return answer;
}

File: 코딩 기초 트레이닝/수 조작하기1-p181926/수 조작하기1.py
# https://school.programmers.co.kr/learn/courses/30/lessons/181926

def solution(n, control):
    answer = n

    dic = {
        'w': 1,
        's': -1,
        'd': 10,
        'a': -10,
    }

    control = list(control)

    for c in control:
        answer += dic[c]

    return answer
